__tests__/app.test.js

```diff
@@ -92,3 +94,240 @@ describe("/api", () => {
+      .post("/api/articles/6/comments")
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.comment.article_id).toBe(6);
+        expect(body.comment.author).toBe("butter_bridge");
+        expect(body.comment.body).toBe(
+          "this could have been avoided if we had known before what we know now"
+        );
+        expect(body.comment.comment_id).toBe(19);
+        expect(typeof body.comment.created_at).toBe("string");
+        expect(body.comment.votes).toBe(0);
+      });
+  });
+
+  test("should return 404 error when non-existent username has been entered", () => {
+    const newComment = {
+      username: "Captain Hindsight",
+      body: "this could have been avoided if we had known before what we know now",
+    };
+
+    return request(app)
+      .post("/api/articles/6/comments")
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.msg).toBe("user does not exist");
+      });
+  });
+  test("should return 404 error when valid but non-existent article_id is entered", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "this could have been avoided if we had known before what we know now",
+    };
+    return request(app)
+      .post("/api/articles/2000000000/comments")
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article does not exist");
+      });
+  });
+  test("should return 400 error when invalid article_id is entered", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "this could have been avoided if we had known before what we know now",
+    };
+    return request(app)
+      .post("/api/articles/maliciousCode/comments")
+      .send(newComment)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+});
+describe("PATCH /api/articles/:article_id", () => {
+  test("should return a 200 status code when patched successfully", () => {
+    const voteChange = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(voteChange)
+      .expect(200);
+  });
+  test("should return updated version of article with votes incremented by specified amount", () => {
+    const voteChange = { inc_votes: 5 };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(voteChange)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.article_id).toBe(3);
+        expect(article.title).toBe(
+          "Eight pug gifs that remind me of mitch"
+        );
+        expect(article.topic).toBe("mitch");
+        expect(article.author).toBe("icellusedkars");
+        expect(article.body).toBe("some gifs");
+        expect(article.created_at).toBe("2020-11-03T09:12:00.000Z");
+        expect(article.votes).toBe(5);
+        expect(article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
```

Comment:
You could consider using toMatchObject https://jestjs.io/docs/expect#tomatchobjectobject
Jest does a lot of the hard work here and the syntax is slightly nicer. It means jest will match (at a minimum) the structure of the object you use in the toMatchObject assertion. This means its open to extension but still requires all the properties you care about right now.